samples/connect-wpf/readme.ipynb

```diff
@@ -59,6 +59,9 @@
         }
       ],
       "source": [
+        "#i \"nuget:I:\\oss\\dotnetinteractive\\artifacts\\packages\\Debug\\Shipping\"\n",
```

Comment:
Referencing a hard-coded path won't work for most people, and we don't want to expect people to build the solution in any case.